langchain4j-elasticsearch/src/main/java/dev/langchain4j/store/embedding/elasticsearch/ElasticsearchEmbeddingStore.java

```diff
@@ -333,32 +310,14 @@ private void addAllInternal(List<String> ids, List<Embedding> embeddings, List<T
         try {
             bulkIndex(ids, embeddings, embedded);
         } catch (IOException e) {
+            log.error("[ElasticSearch encounter I/O Exception]", e);
             throw new ElasticsearchRequestFailedException(e.getMessage());
         }
     }
 
-    private void createIndexIfNotExist(String indexName, Integer dimension) {
```

Comment:
So index is not created automatically now? Is this change intended?

Reply:
That's right. We don't recommend creating an index from the code.
See explanations at https://github.com/langchain4j/langchain4j/pull/712/files#r1701975979